cmd/kaf/consume.go

```diff
@@ -242,6 +245,10 @@ func handleMessage(msg *sarama.ConsumerMessage, mu *sync.Mutex) {
 		}
 	}
 
+	if !strings.Contains(string(dataToDisplay), filter) {
```

Comment:
i'm not sure if `contains` is the right approach. not not check for equality? this might not be obvious to the user..